### ISSUE TITLE
Cyber Discovery Server Meetup

### ISSUE THREAD

**Issue by AdamBromiley** [CyberDiscovery/meta#259]
### *A description of the issue:*
(I know this isn't to do with the structure of the server, but whatever happens it'll have to go through admins so... #meta)
Let's have a server meetup whilst we wait for Elite 2020!!! I'm suggesting we meetup and like chill at a park and go out to eat or something, or raid a cybersec convention. Up for discussion!

### *Proposed Solution:*
Create a poll in #announcements on Birmingham vs. London (two of the most popular locations and both easy to get to), maybe say "vote both options if you can make either", and subsequently have a vote on dates (I'm proposing sometime in the next 3 or 4 months). **Alternatively have one at both locations.**

Not much more planning effort is required, everyone makes their own way there I guess. If we don't have enough people then we can always pick other dates.

**Comment by ThisIs0xBC** [comment]
The problem here is, London people aren't going to be able to justify the 70 quid on train tickets to Birmingham, and Birmingham area people aren't going to be able to justify the 70 quid on train tickets to London. So we should have 2 meetups, one in London and one in Birmingham.

**Comment by AdamBromiley** [comment]
Meet in the middle, Warwick Uni meetup 🗿

**Comment by thebeanogamer** [comment]
> Meet in the middle, Warwick Uni meetup

I suppose that means everyone is equally annoyed?

**Comment by NicholasG04** [comment]
Haha we need Milton Keynes - in the middle :D

**Comment by NicholasG04** [comment]
And I would love to go if you'd let me and I can get there/am around lol

**Comment by thebeanogamer** [comment]
As far as cons are concerned, this is a really good list: https://infosec-conferences.com/

Unfortunately, most cons for this year have passed with the notable exception of [CyberThreat](https://www.cyberthreat2019.com/), but that's very expensive.

**Comment by NicholasG04** [comment]
@thebeanogamer even with the discount I won it's ridiculously priced :c

**Comment by ZomBMage** [comment]
Anywhere besides London / immediately closer to the town I live in would make a meetup infeasible for me. I'm not able to drive independently yet, and neither are many members of the community, and when I was going to Leicester (midlands, for those wondering) to participate in CSC's F2F last year, train tickets would have been in excess of £150 return. London also has the benefit of being a large place with lots to do, and is pretty easily accessible for most of the country. If we weren't crashing a convention, I feel for many people, *especially so* in the south of the country, a meetup anywhere outside of London doesn't make sense, financially or otherwise.

In terms of crashing a convention then, a lot of these can be very expensive, and let's not forget the fact many of them may not be so accommodating for younger people. Also, my mum would have a heart attack at the proposition that i go away to one of these conventions, other people may be able to relate.

If I were to make a suggestion, meeting at one of London's parks would be the easiest option, with plenty of options for travel, food, shopping, and other general entertainment.

**Comment by ThisIs0xBC** [comment]
> London also has the benefit of being a large place with lots to do, and is pretty easily accessible for most of the country

This might be true, but although it is accessible, for anyone above milton keynes it costs a RIDICULOUS amount to get there and back. For me (Worcestershire area) it costs 70 quid for a return ticket at decent times of the day. Not happening. A train ticket from my area to Birmingham however, costs a tenner. We would need two meetups, because people in London are completely seperate from anyone north of Milton Keynes or Oxford due to the insane pricing of train tickets. I know for a fact that barely anyone above those 2 locations would go to a London meetup

**Comment by NateKomodo** [comment]
i can get from Birmingham to London for £10 with my railcard

**Comment by picapi** [comment]
This sounds like an awesome idea, although some consideration might need to be given to the OPSEC of the whole procedure to insure that detailed information about any events or plans remains semi-private, considering the audience for these events with the public nature of the server.

On a side note, there are ways to reduce ticket costs for transport which I'm willing to go into if people wish, although ideally we should minimise these where possible to keep things accessable for all.

**Comment by NateKomodo** [comment]
GUYS WHERE DO YOU LIVE LETS HAVE A HOUSE PARTY SEND ADDRESS YAY

**Comment by AdamBromiley** [comment]
I think if we get a location sorted soon before this idea all gets forgotten it'll be good. Then we can start considering specifically where/when/how. I guess there's going to be people from either side who can't make it, so it might be easier to do both locations. Either way have a vote on them.

**Comment by jmxffcd** [comment]
A̶l̶l̶ ̶e̶x̶p̶e̶n̶s̶e̶s̶ ̶o̶n̶ ̶m̶e̶

Sounds like a cool idea; I wouldn't mind a 30 min train journey to London for it

**Comment by Bentechy66** [comment]
mcr best elite location

**Comment by thebeanogamer** [comment]
> 
> 
> mcr best elite location

But you went to London...

**Comment by Bentechy66** [comment]
mcr best meetup location*

**Comment by NateKomodo** [comment]
if you think 60% of the server is going to come up from london to mcr you are mistaken

**Comment by ThisIs0xBC** [comment]
> if you think 60% of the server is going to come up from london to mcr you are mistaken

And if  you think the other 40% are gonna come down from Bham area to London you are mistaken. Also, not everyone has a railcard like you.

**Comment by jmxffcd** [comment]
Ngl I wouldn't go to Manchester

**Comment by NateKomodo** [comment]
> 
> 
> > if you think 60% of the server is going to come up from london to mcr you are mistaken
> 
> And if you think the other 40% are gonna come down from Bham area to London you are mistaken. Also, not everyone has a railcard like you.

What i was going to propose, but forgot to, was to just have 2 different locations

**Comment by ThisIs0xBC** [comment]
> have 2 different locations

Like I have already said like 3 times lol

**Comment by lightspeedana** [comment]
Hey, I think someone should probably plan this out properly. Have we got dates, locations and things sorted? Then we can probably start thinking of activities.

I personally think London is the best place to meet because it's the most central with best links - I don't think anywhere else has enough people in the vicinity or enough people able to properly go to justify meeting there, and two meetups would be rather excessive. As for activities, if we're inviting and managing a lot of people, laser tag places have great student discounts which I think could work, although once again, a meetup needs a few people actually planning it as too many cooks will spoil the broth.

So TL;DR get a few people planning this, sort out location, sort out best date via voting and then think of cost and activities. I wouldn't mind helping plan this out since I've done a few before and know quite a few CD people so can help in terms of location and logistics etc. if no one else would like to take charge 😄 let's make this happen though, great idea!

**Comment by NicholasG04** [comment]
I agree that two meetups is too much, but there is possibility for a compromise between the north and the south. Also, the West Coast Main Line (Euston-MK-Bham etc) still has 34% off on unidays.

**Comment by NicholasG04** [comment]
@picapi if I'm honest I think your comment is slightly ridiculous. If you care about opsec that much then a meetup probably isn't ideal for you.... and nor is elite.

**Comment by lightspeedana** [comment]
Closing this issue since there's a server for discussion of this now and the meta issue itself is somewhat inactive 😄
